rpm/harbour-watchlist.spec

```diff
@@ -44,8 +44,8 @@ export VERSION_NUMBER=%{version}
 make %{?_smp_mflags}
 
 # >> build post
-echo "Strip the binary!"
```

Comment:
please revert this change - i want to keep that for the releases to have a small binary - i also do remove that when developing locally.

Reply:
Ok. Just did. I'll submit that pull. I'll try to work out why it doesn't work for me. It's kind of common. :) Thanks!